backend/printing/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
```

Comment:
the model and serializer seems unnecessary. Is anything saved to database with it?